src/test/compile-fail/no-implicit-prelude-nested.rs

```diff
@@ -15,6 +15,8 @@
 // fail with the same error message).
 
 #[no_implicit_prelude]
+//~^ WARNING: deprecated
+//~^^ WARNING: deprecated
```

Comment:
Hm ok, we probably want to figure out whether we're deprecating or not before going to great lengths to figure this out, and it's probably fine either way.